tests/jballerina-unit-test/src/test/resources/test-src/javainterop/basic/field_access_mutate_tests.bal

```diff
@@ -16,6 +16,19 @@ function testStaticPrimitiveFieldMutate(int shortValue) {
     return setShortValue(shortValue);
 }
 
+function testStaticPrimitiveFieldMutate2() {
+    boolean bool = true;
+    setBoolean(bool);
+    assertValueEquality(getBoolean(), true);
+}
+
+public type bool true|false|5;
+function testBFiniteToJBooleanCast() {
+    bool arg = true;
+    setFiniteToBoolean(arg);
+    assertValueEquality(getBoolean(), true);
```

Comment:
Shall we use `test:assertEquals` from the `ballerina/test` package

Reply:
Example usage can be seen in static_method_tests.bal : https://github.com/ballerina-platform/ballerina-lang/blob/c0d7162adc2840aa79db18464623656248cf5117/tests/jballerina-unit-test/src/test/resources/test-src/javainterop/basic/static_method_tests.bal#L19